frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <div class="container-fluid">
+    <textarea
+      v-show="false"
+      v-model="value"
+      data-feedback-code-mirror
+    ></textarea>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import T from '../../lang';
+import CodeMirror from 'codemirror';
+import { EditorOptions, languageModeMap, modeList } from './CodeView.vue';
+
+for (const mode of modeList) {
+  require(`codemirror/mode/${mode}/${mode}.js`);
+}
+
+@Component({
+  components: {},
+})
+export default class FeedbackCodeView extends Vue {
+  @Prop() language!: string;
+  @Prop() value!: string;
+  @Prop({ default: false }) enableFeedback!: boolean;
+
+  T = T;
+  mode = languageModeMap[this.language] || languageModeMap['cpp17-gcc'];
```

Comment:
```suggestion
  mode = languageModeMap[this.language] ?? languageModeMap['cpp17-gcc'];
```

siempre que se pueda hay que usar `??` para que cosas como cadenas vacías y `false` no se reemplacen.

Reply:
Listo